### PR TITLE
First draft of IPFS Weekly #22

### PR DESCRIPTION
**Pull request by jennwrites**
Looking for feedback and suggestions on this draft so that we can send it out via TinyLetter (and post on the blog) Tuesday, December 11. 🎄

Thanks!

### CODE REVIEW

File: published/022-2018-dec-11.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
++ [Epicenter.tv featuring Nick Sullivan of Cloudflare, talking about The Internet’s (De)centralized Security Blanket](https://epicenter.tv/episode/264/), Epicenter, 04 Dec 2018
++ [The IPFS Cloud](https://medium.com/pinata/the-ipfs-cloud-352ecaa3ba76), Pinata, 04 Dec 2018
++ [Easy integrations with Textile Thread subscriptions](https://medium.com/textileio/easy-integrations-with-textile-thread-streams-7f1e3ce5c9b4), Textile, 03 Dec 2018
+
+
+## Tools and projects we <3
+*[Awesome IPFS](https://awesome.ipfs.io/) is a community maintained and updated list of projects, tools, or pretty much any things related to IPFS that are totally awesome. To see more, or add yours to the list, visit [Awesome IPFS on GitHub](https://github.com/ipfs/awesome-ipfs).* 
+
++ [Announcing PhysX SDK 4.0, an Open-Source Physics Engine](https://news.developer.nvidia.com/announcing-physx-sdk-4-0-an-open-source-physics-engine/)
```
Comment (victorb) [inline published/022-2018-dec-11.md:35]: Not related to IPFS, even though it's very cool! :)

**Comment by michellebrous** [comment]
@renrutnnej Can we add this job posting to the IPFS newsletter? [Filecoin China BD Program Manager](https://jobs.lever.co/protocol/0f8baa9d-971e-4ca5-9e62-dc63fe7cadd8). Thanks!

**Comment by jennwrites** [comment]
@michellebrous I had planned on putting a line in there at the top for the several job postings at PL right now.

**Comment by jennwrites** [comment]
IPFS Weekly 22 was published:

On the [IPFS Weekly Updates issue](https://github.com/ipfs/ipfs/issues/151#issuecomment-446274875)
Still pending on the blog
And sent to 3012 subscribers [via TinyLetter](https://tinyletter.com/ipfsnewsletter/letters/ipfs-weekly-22)

**Pull request merged**